src/tls.c

```diff
@@ -10750,6 +10853,15 @@ static int TLSX_Write(TLSX* list, byte* output, byte* semaphore,
             case TLSX_USE_SRTP:
                 offset += SRTP_WRITE((TlsxSrtp*)extension->data, output+offset);
                 break;
+#endif
+#ifdef WOLFSSL_QUIC
+            case TLSX_KEY_QUIC_TP_PARAMS:
+                FALL_THROUGH;
+            case TLSX_KEY_QUIC_TP_PARAMS_DRAFT:
+                WOLFSSL_MSG("QUIC transport parameter to write");
+                offset += TLSX_QuicTP_Write((QuicTransportParam*)extension->data,
```

Comment:
For consistency, use a define like QTP_WRITE.

Reply:
Added.